Cargo.toml

```diff
@@ -292,6 +292,7 @@ url = "2.3.1"
 humantime = "2.0.0"
 postcard = { version = "1.0.8", default-features = false, features = ['alloc'] }
 criterion = { version = "0.5.0", default-features = false, features = ["html_reports", "rayon"] }
+fxhash = {package = "rustc-hash", version = "1.1.0"}
```

Comment:
There is no need to rename this package, right?

Reply:
I actually made a mistake: fxhash specifically is currently needed as souper-harvest needs this hash convenience function which is not present in rustc-hash https://docs.rs/fxhash/0.2.1/fxhash/fn.hash.html .

Reply:
I didn't mean undoing the migration, I mean using `rustc-hash = "1.1.0"` instead of `fxhash = {package = "rustc-hash", version = "1.1.0"}`.

Reply:
I chose to rename though to minimise the size of the diff: appreciate if that is considered ugly and so not done

Reply:
> I actually made a mistake: fxhash specifically is currently needed as souper-harvest needs this hash convenience function which is not present in rustc-hash

Inlining the convenience function is fine I think.

> I chose to rename though to minimise the size of the diff: appreciate if that is considered ugly and so not done

Doing the move in a separate commit without renaming would still be easy to review.

Reply:
Have now done this